tests/lib/utils/property-order-test.js

```diff
@@ -313,7 +315,7 @@ describe('determinePropertyType', () => {
         }`
       );
       const node = context.ast.body[0].body.body[0];
-      expect(propertyOrder.determinePropertyType(node, 'component')).toStrictEqual('property');
+      expect(propertyOrder.determinePropertyType(node, 'component', [])).toStrictEqual('property');
```

Comment:
Can you add some unit tests of `determinePropertyType` with various order arrays?